package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.79.2",
+  "version": "2.79.3-alpha-case-submission-fix",
```

Comment:
Note: This will change when it's actually going to be released. This is so I can get a version ready for testing.